src/main/flight/pid.c

```diff
@@ -537,14 +541,13 @@ static FAST_RAM_ZERO_INIT uint16_t dynLpfMin;
 static FAST_RAM_ZERO_INIT uint16_t dynLpfMax;
 #endif
 
-#ifdef USE_D_CUT
-static FAST_RAM_ZERO_INIT float dtermCutPercent;
-static FAST_RAM_ZERO_INIT float dtermCutPercentInv;
-static FAST_RAM_ZERO_INIT float dtermCutGain;
-static FAST_RAM_ZERO_INIT float dtermCutRangeHz;
-static FAST_RAM_ZERO_INIT float dtermCutLowpassHz;
+#ifdef USE_D_MIN
+static FAST_RAM_ZERO_INIT float dMin[XYZ_AXIS_COUNT];
+static FAST_RAM_ZERO_INIT float dMinPercent[XYZ_AXIS_COUNT];
+static FAST_RAM_ZERO_INIT float dMinGyroGain;
+static FAST_RAM_ZERO_INIT float dMinSetpointGain;
+static FAST_RAM_ZERO_INIT float dMinFactor;
```

Comment:
Unless I'm missing something it seems like this shouldn't be a static or even defined here. Isn't the `dMinFactor`  calculated for each axis on each loop? Seems like you only need to instantiate it with:
```
float dMinFactor = 1.0f;
```
Before the section that actually calculates it near the D calc logic.

Reply:
Thank you for your clear instructions, I had tried to do this but didn't get it to work.  Awesome!